allensdk/brain_observatory/nwb/__init__.py

```diff
@@ -385,9 +433,23 @@ def add_stimulus_presentations(nwbfile, stimulus_table, tag='stimulus_epoch'):
             series.fillna('', inplace=True)
             stimulus_table[colname] = series.transform(str)
 
-    stimulus_table = setup_table_for_epochs(stimulus_table, ts, tag)
```

Comment:
@jsiegle @NileGraddis I was also a bit unsure about whether getting rid of this formatting step (which looks to only affect the `timeseries` column) was a kosher move... the previous `timeseries` column produced by `setup_table_for_epochs` didn't exactly make a lot of sense to me so more scrutiny on this would be greatly appreciated.